src/solvers/td3.ml

```diff
@@ -1077,6 +1080,22 @@ module WP =
             in
             VH.add w y d (* intentional add *)
           )
+
+        let finalize ~vh ~reachable =
+          (* retrigger superstable side writes from unknowns that are still reachable *)
+          if incr_verify then (
+            HM.iter (fun x w ->
+                if HM.mem reachable x then
```

Comment:
Same here, `IncrPrune` should already be doing this. If it doesn't happen, then maybe the list of postsolvers needs to be reordered such that `IncrPrune.finalize` is called before this one.